src/tests/JIT/opt/InstructionCombining/NegMulOrDivToConst.cs

```diff
@@ -218,10 +226,29 @@ static int CheckDivNeg()
         }
 
         [MethodImpl(MethodImplOptions.NoInlining)]
-        static int DivNeg11(int a) => -a / 11;
+        static int MulNegCombined(int a, int b)
+        {
+            //ARM64-FULL-LINE: mneg {{w[0-9]+}}, {{w[0-9]+}}, {{w[0-9]+}}
+            return a * b * -1;
+        }
+
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        static int DivNeg11(int a)
+        {
+            //ARM64-FULL-LINE: smull {{x[0-9]+}}, {{w[0-9]+}}, {{w[0-9]+}}
```

Comment:
Just to confirm, this is not related to the PR, right?

Reply:
Nope, it's not related to the PR. Assembly is added for stricter check for an optimised sequence and to make the tests coherent with `MulNegCombined()`.